javascript/modules/client/src/internal/interfaces/context.ts

```diff
@@ -1,5 +1,6 @@
 import { Signer } from "@ethersproject/abstract-signer";
 import { JsonRpcProvider, Networkish } from "@ethersproject/providers";
+import { Options, IPFSHTTPClient } from "ipfs-http-client";
```

Comment:
```suggestion
import { Options as IPFSOptions, IPFSHTTPClient } from "ipfs-http-client";
```

Reply:
I would rename the usage of `Options` into something like `IpfsOptions`. Otherwise the typing becomes ambiguous to someone consuming the library.